### PR TITLE
Remove the "| YOSHITHA" part from the heading

### PR DESCRIPTION
**Pull request by JP109**
Fixed the header, as per issue.

### CODE REVIEW

File: assets/css/styles.css [hunk 1]
```diff
@@ -8,14 +8,14 @@
...
-
+	font-style: italic;
 }
 
 #paragraph{
 	width:70%;
 	font-size:larger;
 	font-weight:lighter;
-	background:rgb(229, 227, 227);
+	background:#D1D1D1;
```
Comment (YoshithaRathnayake) [inline assets/css/styles.css:18]: Undo these changes @JP109

File: assets/css/styles.css [hunk 2]
```diff
@@ -8,14 +8,14 @@
 	margin:10px auto;
 	font-size:30px;
 	font-family: "Edo";
-
+	font-style: italic;
```
Comment (YoshithaRathnayake) [inline assets/css/styles.css:11]: Undo these changes @JP109

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: assets/css/styles.css (hunk 1 above)
Comment (JP109) [reply assets/css/styles.css:18]: Sure

File: assets/css/styles.css (hunk 2 above)
Comment (JP109) [reply assets/css/styles.css:11]: Sure

File: assets/css/styles.css (hunk 1 above)
Comment (JP109) [reply assets/css/styles.css]: Done!

File: assets/css/styles.css (hunk 2 above)
Comment (JP109) [reply assets/css/styles.css]: Done!

**Review by YoshithaRathnayake — APPROVED** [review]
Good Job !
Thank You @JP109
